test/internal/helpers/Maximizer.java

```diff
@@ -48,7 +49,7 @@ public static double modFor(Modifier modifier) {
   }
 
   public static List<Boost> getBoosts() {
-    return net.sourceforge.kolmafia.maximizer.Maximizer.boosts;
+    return new ArrayList<>(net.sourceforge.kolmafia.maximizer.Maximizer.boosts);
```

Comment:
why was this done?

Reply:
Because I wanted to run the maximizer twice and compare the results as part of a test.  Without this if a test wanted to compare boosts under two different sets of conditions the test needed to make its own copy before running again.  It is possible the test that did that has not survived in which case I can revert this but I'm not always a fan of a hiding a mutable list behind an accessor call.

Reply:
okay, seems reasonable